### PR TITLE
support hh:mm:ss or mm:ss as time input

### PR DESCRIPTION
**Pull request by jziolkowski**

### CODE REVIEW

File: smartcut/__main__.py [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 from smartcut.cut_video import smart_cut, VideoSettings, VideoExportMode, VideoExportQuality, AudioExportSettings, AudioExportInfo
 from tqdm import tqdm
 
+def time_to_fraction(time_str_elem):
+    if ':' in time_str_elem:
+        time = None
+        for pattern in ["%H:%M:%S", "%M:%S"]:
+            try:
+                time = datetime.strptime(time_str_elem, pattern)
+            except ValueError:
```
Comment (Secrus) [inline smartcut/__main__.py:15]: replace with `contextlib.supress`

Comment (jziolkowski) [reply smartcut/__main__.py]: done, cool tip

**Comment by skeskinen** [comment]
Thanks, I'm gonna merge this before I make next release. So that the readme is up to date with the binary version.

**Comment by jziolkowski** [comment]
The only issue I see with this naive approach is that it accepts max 23:59:59 as a single timestamp; more that 24hrs are not supported. Do you think it's an issue?

**Comment by skeskinen** [comment]
No, I think if somebody has videos longer than 24h, they can come up with their own solution.
A minor nitpick I had is that I wish you had written your own parsing just to get rid of the try-catch thing. It shouldn't be more than couple lines to split from ':'.
I guess it would also kind of solve the max 24h problem.

But you decide, I'm happy to merge this either way.

**Comment by jziolkowski** [comment]
Ah didn't notice your comment before you merged.

For such simple use-case I thought that writing a custom timestamp parser would be like reinventing the wheel. Agreed, the 24h limit would be no more. If you want, I can make a separate PR for that.

Or at least ping me when someone requests that feature ;)

**Pull request merged**